manage.py

```diff
@@ -0,0 +1,15 @@
+from flask_script import Manager
```

Comment:
I don't think we need this file in this repository. We can run all the migrations from the api project.

Reply:
I've removed this as after review this is only necessary in one repo.